### DISCUSSION TITLE
Passwordless login with passkeys

### DISCUSSION THREAD

**Discussion by mstaicu** [passwordless-id/webauthn#56] [General]
I'm currently onboarding customers using the `@simplewebauthn` package, but I find the process to be a bit too verbose for my use case.

I wish to give customers the ability to register and login to their accounts only by means of Passkeys. I don't plan on supporting passwords or magic link logins because, in my limited understanding of the protocol, Passkeys should provide the ability to login without any user identifier.

Wanting to know if the `webauthn` package under the `passwordless-id` would let me achieve this. My main friction points with the current implementation is that I need to shuffle around and coordinate registration using `jwt`s and then request the user to imput an identifier such as an email address. I think this is redundant and Passkeys solve this.

Are the examples in the repo demonstrating this functionality, both on the server and on the client?

I guess that I'm trying to ask for a clear example because this is a new paradigm shift, at least in my mind, where we receive the credential from the client side and we have to store those against a user. Subsequently, I assume we can pass during authentication an empty array for credentials and leave the authenticator to discover them (?)

I think this is exactly what I'm trying to achieve https://stackoverflow.com/q/73562080

**Comment by dagnelies** [comment]
Hi. I missed this discussion, sorry. That said, it's also unclear to me what's exactly asked/discussed here.

You can random usernames, sure.
